src/Components/BlogPost/BlogPostNavigation.js

```diff
@@ -59,7 +59,7 @@ const BlogPostPreviousLink = Scrivito.connect(({ currentBlogPost }) => {
     .andNot("id", "equals", currentBlogPost.id())
     .andNot("publishedAt", "isGreaterThan", currentDate)
     .order("publishedAt", "desc")
-    .take(1);
```

Comment:
It now needs to be `const olderPost = Scrivito.getClass("BlogPost")`, since `first()` will already return only one item.